### DISCUSSION TITLE
Getting error when trying to paste picture into frame

### DISCUSSION THREAD

**Discussion by alefDev-prog** [ekqt/screenshot#8] [General]
Hi, I don't know if I am doing something wrong but when I copy a screenshot I have recently taken and click on the frame, nothing is happening. When I then open up the console in Chrome's dev-tools, I get an error message stating that there is no valid data on the clipboard. The problem occurs on this link: _https://screenshot-ekqt.vercel.app/_
Here is what it looks like:

![Screenshot 2023-05-16 163609](https://github.com/ekqt/screenshot/assets/114575583/cc67caf4-08fe-47d0-931a-438c2ddaf66a)

**Comment by iamhectorsosa** [comment]
This could be another great addition. A toast to give the user better information of what's happening. 

This error shows because the app reads the clipboard information and if you do not have an image on the clipboard, it aborts the process and logs this error. Are you sure that you have an image on your clipboard when trying this? How did you copy the image to your clipboard? Thanks for the info in advance.

**Comment by alefDev-prog** [comment]
Hi,

What I did was, simply in my file-explorer, I located an image and copied it using ctrl+c. I later checked if the image really was copied by pasting it into a google Docs file, and that worked. However when I then clicked on the frame at the website, it gave me that error. I have tried with several images and I have gotten the same result.

**Reply by iamhectorsosa** [reply]
When you copy an image from your file explorer, you are copying the file itself into the clipboard, not the image data.

The Clipboard API doesn't recognize this as an image type. Google Docs might be uploading the image from the file encountered.

**Reply by iamhectorsosa** [reply]
I would suggest the following. 
1. Open the image on the default preview app or any viewing app of choice in your OS,
2. Right click on the image and click copy
3. Then try to use this copied image in the Screenshot app

**Reply by alefDev-prog** [reply]
Right, now it works, sorry for my mistake. Nevertheless I think it would be a good idea to provide the user with some kind of information if an error occurs, like in my case.

**Reply by iamhectorsosa** [reply]
I'll write up an issue for this. Thanks for the suggestion! Will close the discussion.

**Reply by iamhectorsosa** [reply]
Integrate Toasts #10
